pages/team.md

```diff
@@ -102,38 +118,36 @@ MATE is translated, or partially translated, for 140 languages.
 We'd like to acknowledge the efforts of the following people who
 helped bootstrap MATE in the early days.
```

Comment:
This comment is no longer relevant, since it's moved to the `## Founders` section above